app/controllers/patients_controller.rb

```diff
@@ -52,7 +52,7 @@ def chart
         end
       end
     else
-      raise ActionController::RoutingError.new('Not Found')
+      raise ActionController::RoutingError.new('Could not find patient with id=#{params[:id]}')
```

Comment:
This interpolation won't work because it isn't between double quotes. Plus I'm not sure we need the added detail because the chart route has the patient's id in the params making it very clear which patient is missing.